### PR TITLE
Adds in Peglegs

### PR DESCRIPTION
**Pull request by Stutternov**
## About The Pull Request

Agonized trying to do but whatever, it's here. Peglegs. Sprite looks kinda meh but whatever it's passable and looks similar to the arm's style. If someone wants to make a better one that isn't thrown together in a few seconds of mutilating the arm sprite then be my guest.

Slows the user down for the peglegs, you're able to walk fine and they otherwise act akin to the hands but - as feet.

You're slowed when you got a pegleg, moreso with two, but it's still playable in a decent way even though you WILL notice the slowdown.

ThePainkiller had to do most of the slowdown code because holy fuck WHY CAN'T IT BE EEEASY

## Why It's Good For The Game

Currently there's no way to attach new legs after people de-limb, causing really agonizing rounds if you get a limb cut off or nuggeted which isn't seemingly that uncommon. This fixes that, letting you order new limbs from the merchant.

## Pre-Merge Checklist
- [X] You tested this on a local server.
- [X] This code did not runtime during testing.
- [X] You documented all of your changes.

### CODE REVIEW

File: code/game/objects/items/rogueitems/prostheticarm.dm [hunk 1]
```diff
@@ -59,3 +59,67 @@
 		attach_limb(H)
 		user.visible_message("<span class='notice'>[user] attaches [src] to [H].</span>")
 		return 1
+
+/obj/item/bodypart/l_leg/rproesthetic
+	name = "wooden lleg"
```
Comment (ThePainkiller) [inline code/game/objects/items/rogueitems/prostheticarm.dm:64]:
```suggestion
	name = "wooden leg"
```

File: code/game/objects/items/rogueitems/prostheticarm.dm [hunk 2]
```diff
@@ -59,3 +59,67 @@
...
+		return -1
+	if(user.temporarilyRemoveItemFromInventory(src))
+		attach_limb(H)
+		if(H.pegleg < 1)
+			H.pegleg++
+		user.visible_message("<span class='notice'>[user] attaches [src] to [H].</span>")
+		return 1
+
+/obj/item/bodypart/r_leg/rproesthetic
+	name = "wooden rleg"
```
Comment (ThePainkiller) [inline code/game/objects/items/rogueitems/prostheticarm.dm:96]:
```suggestion
	name = "wooden leg"
```

**Review by ThePainkiller — COMMENTED** [review]
Wow not even a thanks nor a mention despite this being 100% my code, shame on you 😔

File: code/game/objects/items/rogueitems/prostheticarm.dm (hunk 2 above)
Comment (ThePainkiller) [reply code/game/objects/items/rogueitems/prostheticarm.dm:96]: Yes punish people for not examining the leg and realising where it goes

**Comment by Stutternov** [comment]
> Wow not even a thanks nor a mention despite this being 100% my code, shame on you 😔

added it

**Pull request merged**